### DISCUSSION TITLE
yml and documentation for wgpu.h ?

### DISCUSSION THREAD

**Discussion by liskar-dev** [gfx-rs/wgpu-native#442] [Ideas]
This repo [webgpu-headers](https://github.com/webgpu-native/webgpu-headers) has webgpu.h with documentation comments included, as well as `.yml` file. Is there something like this but for `wgpu.h`?

**Comment by ygdrasil-io** [comment]
I created a "wgpu.yml" from scratch to my kotlin binding. I will create a PR when V23.x will be release.
